test/e2e/network/traffic_distribution.go

```diff
@@ -119,164 +138,323 @@ var _ = common.SIGDescribe(feature.TrafficDistribution, framework.WithFeatureGat
 		}
 	}
 
+	// Data structures for representing client and endpoint pods
+	type endpoint struct {
+		node *v1.Node
+		pod  *v1.Pod
```

Comment:
endpoints is overloaded and in this test more confusing, what about `server`

Reply:
OK. (I originally had "server" but I kept wanting to refer to them as endpoints in comments/logs, so I changed it.)